app/src/main/res/menu/menu.xml

```diff
@@ -3,7 +3,7 @@
     xmlns:app="http://schemas.android.com/apk/res-auto">
 
     <item
-        android:title="Search"
+        android:title="Searc"
```

Comment:
We should dig more about UiAutomator for the UI testing and understand how we can avoid this type of issues.